data/json/items/comestibles/mre.json

```diff
@@ -152,6 +152,22 @@
     "description": "A 'Meal Ready to Eat' with a macaroni marinara entree and everything a hungry soldier needs.  The contents will begin to rot once they're removed from this sealed bag.  Activate or disassemble it to get to its contents.",
     "looks_like": "mre_veggy_box"
   },
+  {
+    "id": "mre_sfettuccine_box",
```

Comment:
Isn't this considered one of the [worst MRE's made in recent memory](https://www.youtube.com/watch?v=0TMjKQ0JK6E)?